jobs/redis/templates/config/redis.conf.erb

```diff
@@ -105,9 +105,19 @@ databases 16
 #
 #   save ""
 
-save 900 1
-save 300 10
-save 60 10000
+<% if_p('redis_save_intervals') do
+    redis_save_intervals = p('redis_save_intervals')
+
+    if redis_save_intervals.length > 0
+        redis_save_intervals.each do | command |
```

Comment:
I think we should have two requirements here:

- to not indent the `save` commands
- to keep it readable and somehow consistent

I tried a few ways and didn't like any of them :)

Maybe one alternative could be to skip indentation, something like:

```
<% if_p('redis_save_intervals') do |redis_save_intervals| -%>
<% if redis_save_intervals.length > 0 -%>
<% redis_save_intervals.each do | command | %>
 save <%= command %>
<% end -%>
<% else -%>
save ""
<% end -%>
<% end -%>
```